Command.md

```diff
@@ -0,0 +1,309 @@
+<!-- omit in toc -->
+# Developer guide
+
+This guide helps you get started developing kubi
+
+Make sure you have the following dependencies installed before setting up your developer environment:
+
+ - Git
+ - Jq
+ - Go
+ - In this context we will use a kind cluster for the local deployment of Kubi : cluster kind {1.23 or 124}
+
+## Deploy kubi  
+
+ - Get the source code https://github.com/ca-gip/kubi
+  
+ - install go  https://go.dev/
+    
+ - Create kind cluster with version 1.24-1.26 https://kind.sigs.k8s.io/docs/user/quick-start/
+  
+ - kubi private key
+ ```
+ openssl ecparam -genkey -name secp521r1 -noout -out /tmp/ecdsa-key.pem
+ ```
+  
+ - kubi public key
+ ```
+ openssl ec -in /tmp/ecdsa-key.pem -pubout -out /tmp/ecdsa-public.pem
+ ```
+  
+ - install CFSSL tools
+ ```
+   VERSION=$(curl --silent "https://api.github.com/repos/cloudflare/cfssl/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
Couldn't this be containerized to avoid installing binaries into the host?

Reply:
should i replace this with a script ?